shared/templates/rsyslog_logfiles_attributes_modify/ansible.template

```diff
@@ -0,0 +1,70 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = medium
+
+{{{ ansible_instantiate_variables(PARAMETER) }}}
+
+- name: "Set rsyslog logfile configuration facts"
+  set_fact:
+    rsyslog_etc_config: "/etc/rsyslog.conf"
+
+# * And also the log file paths listed after rsyslog's $IncludeConfig directive
+#   (store the result into array for the case there's shell glob used as value of IncludeConfig)
+- name: "Get IncludeConfig directive"
+  shell: |
+    set -o pipefail
+    grep -e '$IncludeConfig' {{ rsyslog_etc_config }} | cut -d ' ' -f 2 || true
+  register: rsyslog_old_inc
+  changed_when: False
+
+- name: "Get include files directives"
+  shell: |
+    set -o pipefail
+    grep -oP '^\s*include\s*\(\s*file.*' {{ rsyslog_etc_config }} |cut  -d"\"" -f 2 || true
+  register: rsyslog_new_inc
+  changed_when: False
+
+- name: "Aggregate rsyslog includes"
+  set_fact:
+    include_config_output: "{{ rsyslog_old_inc.stdout_lines + rsyslog_new_inc.stdout_lines }}"
+
+- name: "List all config files"
+  find:
+        paths: "{{ include_config_output | list | map('dirname') }}"
```

Comment:
The indentation is not aligned here. Could you adjust it, please?

Reply:
:+1: should be handled in [7d1d759](https://github.com/ComplianceAsCode/content/pull/9789/commits/7d1d75945b832a8789227681ddee31851ba79152)